status/reporter_github.go

```diff
@@ -537,7 +537,8 @@ func generateGithubCommitState(state intgteststat.IntegrationTestStatus) (string
 
 // Detect if GitHubReporter can be used
 func (r *GitHubReporter) Detect(snapshot *applicationapiv1alpha1.Snapshot) bool {
-	return metadata.HasLabelWithValue(snapshot, gitops.PipelineAsCodeGitProviderLabel, gitops.PipelineAsCodeGitHubProviderType)
+	return metadata.HasLabelWithValue(snapshot, gitops.PipelineAsCodeGitProviderLabel, gitops.PipelineAsCodeGitHubProviderType) ||
```

Comment:
super micro-optimization would be to start checking annotation first.

Reply:
Was just thinking the same thing :D